src/troute-nwm/src/nwm_routing/output.py

```diff
@@ -194,7 +194,10 @@ def nwm_output_generator(
 
     if test:
         flowveldepth.to_pickle(Path(test))
-    
+        
+        nudge = results[0][8]
+        usgs_positions_id = results[0][3][0]
+        nhd_io.write_flowveldepth_netcdf('../LowerColorado_TX/output', flowveldepth, nudge, usgs_positions_id, t0)
```

Comment:
it'd be good to make the `filepath` here a user specified input parameter in the configuration file.